src/components/Pricing/PricingSlider/pricingSliderLogic.ts

```diff
@@ -91,6 +100,21 @@ export const pricingSliderLogic = kea<pricingSliderLogicType>({
                 setSessionRecordingInputValue: (_: null, { value }: { value: number }) => value,
             },
         ],
+        featureFlagSliderValue: [
+            null,
+            {
+                setFeatureFlagSliderValue: (_: null, { value }: { value: number }) => value,
+                setFeatureFlagInputValue: (_: null, { value }: { value: number }) => inverseCurve(value * 1000000),
+            },
+        ],
+        featureFlagInputValue: [
+            1,
+            {
+                setFeatureFlagSliderValue: (_: null, { value }: { value: number }) =>
+                    Math.round(sliderCurve(value) / 1000000),
+                setFeatureFlagInputValue: (_: null, { value }: { value: number }) => value,
+            },
+        ],
```

Comment:
Again feels like this could be dynamic. But understand if that's not a project you want to take on right now.

Reply:
Thought the same thing, but yes, it will have to come later.